src/polyhedron.jl

```diff
@@ -199,7 +199,7 @@ end
 #end
 _getrepfor(p::Polyhedron, ::Polyhedra.HIndex, status::Symbol) = getinem(p, status)
 _getrepfor(p::Polyhedron, ::Polyhedra.VIndex, status::Symbol) = getextm(p, status)
-function Polyhedra.isredundant(p::Polyhedron, idx::Polyhedra.Index; strongly=false, cert=false, solver=nothing)
+function Polyhedra.isredundant(p::Polyhedron, idx::Polyhedra.HIndex; strongly=false, cert=false, solver=nothing)
```

Comment:
This is no longer needed with the latest release of Polyhedra

Reply:
ack

Reply:
```suggestion
function Polyhedra.isredundant(p::Polyhedron, idx::Polyhedra.Index; strongly=false, cert=false, solver=nothing)
```

Reply:
[This](https://github.com/JuliaPolyhedra/Polyhedra.jl/blob/29292ad468ece57abe6974d06a6dbc3eeaadfbf0/src/redundancy.jl#L22-L24) seems a bit code-smelly to me.  I don't think the interface library should be holding back from defining a reasonable partial-specialization of `isredundant` just for the sake of saving some boilerplate in a specific backend.

Reply:
We actually do that a lot in MutableArithmetics and MathOptInterface. When you define a default implementation for a function, it's best to have the signature as wide as possible (in fact, the signature can just be the same as for the docstring), redirect to a `..._fallback` function where you can do any signature you like without any risk of ambiguity since no other package should add methods to `..._fallback`.
Here we did it once we noticed the issue with LRSLib which seems a bit of a hack but we should just do that automatically for all functions that defines different methods as default instead of just one.

Reply:
Ah ok.  That seems like a reasonable pattern, thanks for explaining.  I might PR upstream to rephrase the comment in a way that doesn't sound specific to LRSLib :slightly_smiling_face: